docs/react-native-install.md

```diff
@@ -152,7 +152,16 @@ Note 2: May need to Install `Node.js` if your `npm` command isn't working which
 
 ![goodYarnInstall](images/react_native/yarnInstallGood.png)
 
-## How to Install Node.js on Windows
+# Time to Run the Demo on Android
+
+Now that we've sucesfully installed the required tools and setup. In the directory of `theta-client\demos\demo-react-native` use the command `yarn run android` to start your app in an Android emulator.
+
+**In Progress**...
```

Comment:
It's unclear to me what "In Progress" means.

Reply:
Going to be updating the "In Progress" with more information in my new pull request

Reply:
OK, understood. Thanks for explaining.